### DISCUSSION TITLE
Add `clear` method

### DISCUSSION THREAD

**Discussion by TomJGooding** [davep/textual-canvas#1] [Ideas]
Hi Dave! This library really simplifies drawing inside Textual!

I know this is a hobby project you haven't touched in a little while, but adding a `clear` method for the Canvas might be a useful feature.

**Comment by davep** [comment]
Only hasn't been touched because it's handled all my needs so far and nobody's asked for or suggested anything else for it; so that's a good ask.

Yup, makes perfect sense to add that.

**Reply by TomJGooding** [reply]
Fair enough! I was couching my suggestion a bit as I'm probably enough of a nuisance in the Textual repo...!

**Reply by davep** [reply]
You've lived a very charmed life if you think your helpful contributions to Free Software count as a "nuisance". :-P

**Comment by davep** [comment]
Done: https://github.com/davep/textual-canvas/issues/2#issuecomment-1636888455

**Comment by davep** [comment]
Now released to PyPi: https://pypi.org/project/textual-canvas/0.2.0/
